test/models/refTests.ts

```diff
@@ -0,0 +1,79 @@
+import * as mongoose from 'mongoose';
+
+import { arrayProp, prop, Ref, Typegoose } from '../../src/typegoose';
+
+export class RefTestBuffer extends Typegoose {
+  @prop() public _id: mongoose.Schema.Types.Buffer;
```

Comment:
it would be good if we could keep a consistent class style
so i ask, could you change this (and the following classes) to use the following style?
```ts
@prop()
public _id; // separate line
```